spec/requests/api/go_cardless/go_cardless_spec.rb

```diff
@@ -246,7 +246,9 @@
                 last_name: 'Sanders',
                 akid: '123.456.789',
                 source: 'fb',
-                user_en: 1
+                user_en: 1,
+                user_express_cookie: 0,
+                user_express_account: 0
```

Comment:
This might be an issue. GoCardless donations should not update the custom user fields - the fact that someone is enrolled for Braintree custom donations and then does a GoCardless donation doesn't mean they're no longer an express donor.